data/json/items/crossbows.json

```diff
@@ -1,110 +1,225 @@
 [
   {
     "type": "AMMO",
-    "id": "bolt_wood",
```

Comment:
I'm not terribly versed in how the game resolves ids from json into objects but wont changing this id break existing items in the game? I assume thats why you didnt change the ids of the arrows

Reply:
That entry still exists, I just apparently shifted it further down the file and Github didn't pick up on that. Since the id still exists it should be perfectly fine, placement in the file has no effect to my knowledge.

Reply:
Those things are always hard to see. Thanks for the clarity

Reply:
The entire file is fairly wonky and there's an argument to be made for shifting the existing IDs to new ones, but that's not a very high priority for me at the moment.